moddingway/database/notes_database.py

```diff
@@ -0,0 +1,120 @@
+from dataclasses import dataclass
+from . import DatabaseConnection
+from .models import Note
+from typing import List
+
+
+@dataclass
+class Note_Display:
+    note_id: int
+    content: str
+    created_by: int
+    last_editor: int
+
+
+def from_db_row(row: tuple) -> "Note_Display":
```

Comment:
nit: this should be Note_Display, not a string

Reply:
also nit: function names usually start with a verb